src/modules/confirmation/components/TransactionBalance.vue

```diff
@@ -23,48 +23,77 @@
             height="17"
           />
         </v-col>
-        <v-col class="py-0 my-0 text-right" cols="5">
-          <span> - {{ value }} </span>
+        <v-col
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
+          cols="12"
+          sm="4"
+        >
+          <span class="mr-1">- {{ value }}</span>
           <span class="searchText--text font-weight-medium">{{
             currency
-          }}</span></v-col
+          }}</span>
+        </v-col>
+        <v-col
+          v-if="isEthNetwork"
+          class="d-flex align-center pl-6"
+          :class="$vuetify.breakpoint.smAndUp ? 'justify-end' : 'justify-start'"
```

Comment:
change this to: class="justify-start justify-sm-end"

Reply:
fixed